README.md

```diff
@@ -13,7 +13,7 @@
 ##### Fedora
 Install ruby and development tools (preferably in a [toolbx]*(https://containertoolbx.org) or [distrobox](https://github.com/89luca89/distrobox) container):
 ```bash
-sudo dnf install ruby rubygems ruby-devel gcc gcc-c++ rpm-build
+sudo dnf install ruby rubygems ruby-devel rubygem-jekyll gcc gcc-c++ rpm-build
```

Comment:
I think I'm going to reorganize/rework the README setup steps to be more clear. I would like to minimize the number of system packages installed via dnf, and I believe the step below to install jekyll via gem works